### DISCUSSION TITLE
any way to set sslVerify false on issue any git command

### DISCUSSION THREAD

**Discussion by cmingxu** [gitpython-developers/GitPython#1599] [Q&A]
Hi team, I am new to gitpython, and I am looking a way to disable sslVerify upon issue a git command, don't know how to pass parameters, thanks

**Comment by Byron** [comment]
You probably want to change the `GIT_SSH_COMMAND` to your own script to control how `ssh` is invoked, as [described in the docs](https://gitpython.readthedocs.io/en/stable/tutorial.html#handling-remotes).

Here is an example to show the general idea:

```python
ssh_cmd = 'ssh -i id_deployment_key'
with repo.git.custom_environment(GIT_SSH_COMMAND=ssh_cmd):
    repo.remotes.origin.fetch()
```
